Dockerfile

```diff
@@ -12,23 +12,28 @@ LABEL io.k8s.description="Archival code service" \
       io.openshift.min-memory="64Mi"
 
 WORKDIR /home/repour
-ENTRYPOINT ["./setup.sh", "-", "./pid1.py", "./au.py", "python3", "-m", "repour"]
+ENTRYPOINT ["sudo", "-E", "-u", "repour", "./pid1.py", "./au.py", "python3", "-m", "repour"]
 CMD ["run-container"]
 
 RUN cd / && \
     groupadd -rg 1001 repour && \
     useradd -rm -u 1001 -g repour repour && \
     chmod og+rwx /home/repour/ && \
     echo "tsflags=nodocs" >> /etc/dnf/dnf.conf && \
-    dnf install -y bsdtar python3 java-headless git subversion mercurial nss_wrapper gettext sudo ca-certificates && \
+    dnf install -y bsdtar python3 java-headless git subversion mercurial nss_wrapper \
+        gettext sudo ca-certificates asciidoc && \
     dnf clean all && \
     echo -ne '\n\tStrictHostKeyChecking no\n\tPreferredAuthentications publickey\n\tIdentityFile /mnt/secrets/repour/repour\n\tControlMaster auto\n\tControlPath /tmp/%r@%h-%p\n\tControlPersist 300\n' >> /etc/ssh/ssh_config
 
-COPY ["venv/container.txt", "container/pid1.py", "container/au.py", "script/*", "/home/repour/"]
+COPY [".git/", "/home/repour/.git"] # needed for make-readme.sh, deleted later
```

Comment:
Also, regarding the comment, you can't delete stuff from an image once it's been added and the layer closed. For example, this is why there's a `dnf clean all` in the same RUN command as `dnf install`.